WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -877,6 +904,19 @@ private extension NotificationsViewController {
             resetApplicationBadge()
         }
     }
+
+    func updateMarkAllAsReadTintColor() {
+        guard let notes = tableViewHandler.resultsController.fetchedObjects as? [Notification] else {
+            return
+        }
+
+        let isEnabled = !notes.filter {
+            !$0.read
+        }.isEmpty
```

Comment:
I think something like this is a little more efficient:
`let isEnabled = notes.first { !$0.read } != nil`

Using `.first`, it just has to find one instead of finding them all.

Reply:
Definitely. Nice catch!